### PR TITLE
Update User Research page “getting-informed-consent-for-user-research”

### PR DESCRIPTION
**Pull request by johnwaterworth**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by johnwaterworth** [comment]
Adding guidance on who counts as a participant and should be asked for their consent.

File: src/user-research/getting-informed-consent-for-user-research.md [hunk 1]
```diff
@@ -7,11 +7,29 @@ You must have informed consent from participants before you start any research a
 
 This guidance explains how we collect and keep evidence of informed consent at dxw. It builds on the general guidance on [getting informed consent](https://www.gov.uk/service-manual/user-research/getting-users-consent-for-research) in the GOV.UK Service Manual.
 
-## Using dxw or client consent materials and processes
+Note that this guidance applies to all dxw staff, dxw friends, and partner and client staff doing research activities, like interviews, visits, surveys and tests.
```
Comment (ZuleimaMR) [inline src/user-research/getting-informed-consent-for-user-research.md:10]: This sentence refers to any type of research, as in business and technical research also or only user research? From the moment i started at dxw it was evident in my team that the business analyst does research. Should this sentence then be more specific for starters like me?

**Review by ZuleimaMR — CHANGES_REQUESTED** [review]
I have reviewed this section and left one comment only. Everything else I had reviewed already when we were asked to do so in the google drive version and seeing this version here i cannot see anything that immediately needs correction or improvements. I had provided suggestions in the google drive version that I can see some were adopted and other not. So, at this moment, I would only need my comment to be addressed and then I can revisit to approve.

File: src/user-research/getting-informed-consent-for-user-research.md (hunk 1 above)
Comment (johnwaterworth) [reply src/user-research/getting-informed-consent-for-user-research.md:10]: The guidance applies to business analysts, or anyone else who is doing the kinds of research it describes.

**Review by FrancesU2023 — APPROVED** [review]

**Pull request merged**